db/migrations/mysql/000003_create_commands.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS Commands (
+    Id varchar(26) NOT NULL,
+    Token varchar(26) DEFAULT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UpdateAt bigint(20) DEFAULT NULL,
+    DeleteAt bigint(20) DEFAULT NULL,
+    CreatorId varchar(26) DEFAULT NULL,
+    TeamId varchar(26) DEFAULT NULL,
+    `Trigger` varchar(128) DEFAULT NULL,
+    Method varchar(1) DEFAULT NULL,
+    Username varchar(64) DEFAULT NULL,
+    IconURL text,
+    AutoComplete tinyint(1) DEFAULT NULL,
+    AutoCompleteDesc text,
+    AutoCompleteHint text,
+    DisplayName varchar(64) DEFAULT NULL,
+    Description varchar(128) DEFAULT NULL,
+    URL text,
+    PluginId text,
```

Comment:
We should remove this from the create table as it's added right after in the alter

Reply:
Full disclosure on this, i was trying to mimic what gorp + upgrade is doing. But happy to drop that column from here